src/main/resources/api/service_openapi.yaml

```diff
@@ -191,6 +194,228 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{id}/resources/references/datarepo/snapshots:
+    parameters:
+    - $ref: '#/components/parameters/Id'
```

Comment:
not for this PR: I really wish this was named `workspaceId` instead of the totally-ambiguous `id`

Reply:
it really should be, that's tripped people up in client tests before when `id` gets mixed up between workspaces and references.